build/source/engine/read_attrb.f90

```diff
@@ -349,6 +349,20 @@ subroutine read_attrb(attrFile,nGRU,attrStruct,typeStruct,idStruct,err,message)
   end select ! select variable
 
  end do ! (looping through netcdf local attribute file)
+ 
+ ! ** now handle the optional aspect variable if it's missing
+ varIndx = get_ixAttr('aspect')
+ ! check that the variable was not found in the attribute file
+ if(.not. checkAttr(varIndx)) then
+   write(*,*) NEW_LINE('A')//'INFO: aspect not found in the input attribute file, continuing ...'//NEW_LINE('A')
+
+   do iGRU=1,nGRU
+    do iHRU = 1, gru_struc(iGRU)%hruCount
+     attrStruct%gru(iGRU)%hru(iHRU)%var(varIndx) = -1._dp      ! populate variable with out-of-range value, used later
```

Comment:
Could this be changed to a missing value placeholder from `engine/nrtype.f90` so that the intention is clearer?

Reply:
done